package.json

```diff
@@ -4,13 +4,12 @@
   "description": "Isomorphic React boilerplate",
   "main": "index.js",
   "scripts": {
-    "start": "node index",
+    "start": "APP_ENV=development node index",
```

Comment:
Oh man, why didn't we think to already do that? Nice catch. Also can you update the documentation -- I think it explains how to run it the 'manual' way.